testmods/src/main/resources/assets/legacy-rewoven-api/lang/en_US.lang

```diff
@@ -0,0 +1,4 @@
+# Keybindings
```

Comment:
you forgot to derewoven this file

Reply:
No that's intentional.

Reply:
ok

Reply:
Wait there's supposed to be a second one in the right folder.

Reply:
That was meant to be left in there, but there should also be a proper one.
Facking IDEA facking deleting my shit. ;-;

Reply:
concern

Reply:
Should be good now.